clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/InternalLocalBootstrappingVeniceChangelogConsumer.java

```diff
@@ -376,7 +417,10 @@ public void run() {
                 VeniceChangeCoordinate
                     .convertVeniceChangeCoordinateToStringAndEncode(state.getValue().currentPubSubPosition));
           } catch (IOException e) {
-            // TODO log error
+            // TODO:
```

Comment:
Can probably remove TODO

Reply:
Sure, I'll remove when everything is good with unit tests later.